Source/AhriAndBear/AI/BtTaskClearKey.h

```diff
@@ -0,0 +1,21 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "BehaviorTree/BTTaskNode.h"
+#include "BtTaskClearKey.generated.h"
+
+/**
+ * 
+ */
+UCLASS()
+class AHRIANDBEAR_API UBtTaskClearKey : public UBTTaskNode
+{
+	GENERATED_BODY()
+
+	UPROPERTY(Category = "AI", EditAnywhere)
+		FName KetToClear;
```

Comment:
Should this be `KeyToClear`? We should probably also comment this so other devs can know its purpose.

Reply:
Yeah. Sorry Typo I'll update it